Sources/MapboxDirectionsCLI/ProcessCommand.swift

```diff
@@ -1,38 +1,14 @@
 
 import Foundation
 import MapboxDirections
-import SwiftCLI
-
 
 private let BogusCredentials = DirectionsCredentials(accessToken: "pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede")
 
-class ProcessCommand<ResponceType : Codable, OptionsType : DirectionsOptions > : Command {
+class ProcessCommand<ResponceType : Codable, OptionsType : DirectionsOptions > {
```

Comment:
Can we rename the class as it looks like a `ParsableCommand` but not like that anymore